content/docs/getting-started/hello-world.md

```diff
@@ -175,27 +141,61 @@ different pros and cons. In this example, we will use the default executor of th
 [Tokio runtime][rt].
 
 ```rust
-# #![deny(deprecated)]
-# extern crate tokio;
-# extern crate futures;
-#
-# use tokio::prelude::*;
-# use futures::future;
-# fn main() {
-# let client = future::ok(());
-println!("About to create the stream and write to it...");
-tokio::run(client);
-println!("Stream has been created and written to.");
-# }
+use tokio::io::AsyncWriteExt;
+use tokio::net::TcpStream;
+use tokio::runtime::Runtime;
+
+fn main() {
+    let addr = "127.0.0.1:6142";
+    let client = TcpStream::connect(addr);
+
+    let future = async move {
+        if let Ok(mut stream) = client.await {
+            println!("created stream");
+
+            let result = stream.write_all(b"hello world\n").await;
+
+            println!("wrote to stream; success={:?}", result.is_ok());
+        }
+    };
+
+    let rt = Runtime::new().unwrap();
+    println!("About to create the stream and write to it...");
+    rt.block_on(future);
+    println!("Stream has been created and written to.");
```

Comment:
Since these are "getting started" docs, maybe we should just use the batteries-included `#[tokio::main]` attribute, rather than creating a `Runtime` manually (more of an advanced API). I notice that the previous version used `tokio::run`, which, IMO, is more or less the tokio 0.1 equivalent of `#[tokio::main]`, so perhaps we should continue using the simpler API?